src/app/record/components/work/work.component.html

```diff
@@ -443,7 +413,7 @@ <h3 class="orc-font-body-small">
           </ng-container>
         </ng-container>
       </ng-container>
-      <ng-container *ngIf="!togglzWorksContributors && showCitation">
+      <ng-container *ngIf="showCitation">
```

Comment:
The enable mode of `togglzWorksContributors` never shows this container. 
I think this container should be removed